src/InstanceSerialization/InstanceMetadata.php

```diff
@@ -92,13 +92,22 @@ public function __construct(string $instance) {
 
       $this->as_float32[] = strpos($property->getDocComment(), '@kphp-serialized-float32') !== false;
 
+      // get type either from @var or from php 7.4 field type hint
+      $type = '';
       preg_match('/@var\s+([^\n]+)/', $property->getDocComment(), $matches);
-      assert(count($matches) > 1);
-      $type     = (string)$matches[1];
+      if (count($matches) > 1) {
+        $type = (string)$matches[1];
```

Comment:
Casting to string is not necessary, all `$mathes` values are strings.

Reply:
Yes, but... Whyever, PhpStorm doesn't think so in its internals 😀
![image](https://user-images.githubusercontent.com/67757852/110942539-8110ca80-836c-11eb-833d-392bb96956fa.png)
So, I'll leave this cast for now